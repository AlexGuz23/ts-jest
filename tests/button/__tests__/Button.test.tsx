declare var jest, describe, it, expect, require;

import * as React from 'react';
const renderer = require('react-test-renderer');

import { Button, BadButton } from '../Button';

it('Button renders correctly', () => {
  const tree = renderer.create(<Button>hi!</Button>).toJSON();
  expect(tree).toMatchSnapshot();
});

xit('BadButton should throw an error on line 18', () => {

  renderer.create(<BadButton>hi!</BadButton>).toJSON();

});
