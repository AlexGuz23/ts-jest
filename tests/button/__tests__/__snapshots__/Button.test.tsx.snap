exports[`test Button renders correctly 1`] = `
<div
  className="button">
  hi!
</div>
`;
